src/context/config-context.tsx

```diff
@@ -8,8 +8,8 @@ export const ConfigContext = createContext({
   setConfigExpanded: (value: boolean) => {}
 })
 
-export const ConfigProvider = ({ children, expanded = isLoadedInIframe }: { children: JSX.Element[] | JSX.Element, expanded?: boolean }): JSX.Element => {
-  const [isConfigExpanded, setConfigExpanded] = useState(expanded)
+export const ConfigProvider = ({ children }: { children: JSX.Element[] | JSX.Element, expanded?: boolean }): JSX.Element => {
+  const [isConfigExpanded, setConfigExpanded] = useState(isConfigPage(window.location.hash))
```

Comment:
could be updated to loaded dynamically but I don't think it would give us much for the effort